test/index.spec.js

```diff
@@ -730,6 +738,32 @@ describe('onBuild()', () => {
     expect(existsSync(path.join('.netlify', 'edge-functions', 'ipx', 'index.ts'))).toBeTruthy()
   })
 
+  test('generates edge-functions manifest', async () => {
+    await moveNextDist()
+    await nextRuntime.onBuild(defaultArgs)
+    expect(existsSync(path.join('.netlify', 'edge-functions', 'manifest.json'))).toBeTruthy()
+  })
+
+  test('generates generator field within the edge-functions manifest', async () => {
+    await moveNextDist()
+    await nextRuntime.onBuild(defaultArgs)
+    const manifestPath = await readJson(path.resolve('.netlify/edge-functions/manifest.json'))
+    const manifest = manifestPath.functions.sort()
+    
+    expect(manifest).toMatchSnapshot()
```

Comment:
I think we're overusing snapshots in most of our tests, for me it's typically the last resort.

I'd use this instead:

```ts
expect(manifest).toEqual(
  expect.arrayContaining([
    expect.objectContaining({
      generator: '@netlify/next-runtime@1.0.0'
    })
  ])
)
```

Applies also to the test below.

You then can also drop the `sort()` as the order is not relevant anymore.

Some reading: https://www.emgoto.com/jest-partial-match/

Reply:
This is really helpful, thanks Lennart!!